### ISSUE TITLE
Try to handle circa dates better in Decade processing function

### ISSUE THREAD

**Issue by wickr** [OregonDigital/oregondigital#830]
Some(all?) circa dates aren't handled by the Decade processor.

Try stripping out:
ca.
circa
#### circa  (OHBA has these)

before sending the date through.

**Comment by jsimic** [comment]
We've been trying to use the earliestDate and latestDate predicates for circa dates instead. When we catch them, that is.

**Comment by wickr** [comment]
Good point. I don't think most OSU collections have them (yet). It would be nice to have the Decade code pull from those too if they exist.
